pkg/logutil/rate.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package logutil
+
+import (
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+// TrivialRater is a trivial rate tracer.
+// It doesn't record any time sequence, and always
+// return the average speed over all the time.
+// TODO: replace it with Prometheus.
+type TrivialRater struct {
+	start   time.Time
+	current uint64
+}
+
+// NewTrivialRater make a trivial rater.
+func NewTrivialRater() TrivialRater {
+	return TrivialRater{
+		start:   time.Now(),
+		current: 0,
+	}
+}
+
+// Success adds n success units for the rater.
+func (r *TrivialRater) Success(n uint64) {
+	atomic.AddUint64(&r.current, n)
+}
+
+// Rate returns the rate over all time, in the given unit.
+func (r *TrivialRater) Rate(unit time.Duration) float64 {
```

Comment:
are we going to use any units other than `time.Second`?

Reply:
Aha, maybe... 🤔

With the idea "it might be useful in the future", I added this argument. However seems it should be a constant for now.🙃